CHANGELOG.md

```diff
@@ -8,10 +8,17 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [2.1.0] - 2019-01-30
+
 ### Added
```

Comment:
Missing: Show a thumbnail of the video on the Dashboard when it is ready to play